integreat_cms/cms/fixtures/test_data.json

```diff
@@ -413,6 +415,7 @@
       "text_direction": "RIGHT_TO_LEFT",
       "primary_country_code": "ab",
       "table_of_contents": "محتويات",
+      "socialmedia_webapp_title": "Integreat | Web-App | Lokale Informationen für Dich",
```

Comment:
Are you just reffering to the tests or also in the live system? We also changed the remaining part, as it does make sense for e.g. Malte to have a different title.

Reply:
Also for production, I think in this case I was referring to the translated part, so I wondered whether it really makes sense to show the German "Web-App | Lokale Informationen für Dich" as fallback for Arabic content or whether we should rather just stick with the branding title without the rest... I mean in most cases we will have the correct translation stored in the language model, I was just thinking about cases where it might not be.

Reply:
@dkehne, I guess? We've had quite a long discussion about this, already. 

We could also just show the branding title as fallback for e.g. Arabic.